src/createStore.js

```diff
@@ -123,7 +123,13 @@ export default function createStore(reducer, initialState) {
       isDispatching = false
     }
 
-    listeners.slice().forEach(listener => listener())
+    listeners.slice().forEach(listener => {
+      // Check if subscription still exists (#1180)
+      if (listeners.includes(listener)) {
```

Comment:
This would assume user has ES6 polyfill (we don't currently depend on it). Also AFAIK `includes` can potentially be O(n) so it's bad to call in a loop.